### ISSUE TITLE
Wrap the global gapi and google objects into a module

### ISSUE THREAD

**Issue by begedin** [coderly/ember-gdrive#38]
Currently, we rely heavily on the global gapi and google objects inside ember-gdrive. This throws jshint warnings, so we also have to rely on `/*global gapi, google*/` to get rid of these. Finding a proper way to wrap these objects into exports would make the code cleaner (and possibly more reliable).

**Comment by venkatd** [comment]
A simple way could be to create a file that does something to the effect of `export default window.gapi`

**Comment by begedin** [comment]
Yeah, that's pretty much all that's needed, but I was thinking a step further.

Something along the lines of combining that and the `addon\lib\loader` to make it as "ember-like" as possible.

**Comment by venkatd** [comment]
I'm interested to hear more on this idea. I like the strategy of wrapping the gapi loading capabilities with Ember's loading capabilities.

I'd do prefer that we prioritized the experience of creating a hello world app in ember-gdrive. IMO the other tasks--in particular ones that improve the code quality--can be pushed to later.

**Comment by begedin** [comment]
Yeah, I agree. Everything works as is, so there's little urgency to try and improve it now. Mind if I add a "low-priority/backlog" label or milestone for this type of stuff?

**Comment by venkatd** [comment]
That's fine. As a rule of thumb, we can create tasks for anything that we might possibly do. That way we don't have to try to keep track of things in our head.

So strategy is
- Dump everything into a list
- Prioritize the list
- Work from most to least important
